atomicapp/applogging.py

```diff
@@ -27,15 +27,16 @@
 class customOutputFormatter(logging.Formatter):
     """
     A class that adds 'longerfilename' support to the logging formatter
-    This 'longerfilename' will be filename + parent dir.
+    This 'longerfilename' will be path/to/file.py after the root atomicapp
+    folder.
     """
 
     def format(self, record):
 
         # Add the 'longerfilename' field to the record dict. This is
         # then used by the Formatter in the logging library when
         # formatting the message string.
-        record.longerfilename = '/'.join(record.pathname.split('/')[-2:])
+        record.longerfilename = record.pathname.split(customOutputFormatter.path)[-1].lstrip('/')
```

Comment:
@dustymabe addressed the thing you mentioned about using `-1`, this is really brilliant and failsafe, nice suggestion!